### PR TITLE
Add whatsapp.net frontend and backend

### LINKED ISSUE #202: +60177390381

### PR DESCRIPTION
**Pull request by eozturk1**
Adding whatsapp.net backend and frontend to the proxy configuration to support whatsapp.net traffic including media.

### CODE REVIEW

**Comment by eozturk1** [comment]
I will also update the `docker-compose.yml` file with these ports.

File: README.md [hunk 1]
```diff
@@ -104,6 +104,7 @@ Depending on the scenario in which you utilize your proxy, the proxy container e
 1. 80: Standard web traffic (HTTP)
 2. 443: Standard web traffic, encrypted (HTTPS)
 3. 5222: Jabber protocol traffic (WhatsApp default)
+4. 587 or 7777: *.whatsapp.net traffic including media
```
Comment (slawlor) [inline README.md:107]: nit: Specify HTTP(S) traffic?

File: proxy/src/proxy_config.cfg [hunk 2]
```diff
@@ -80,6 +80,19 @@ frontend haproxy_v4_xmpp
...
+  #PUBLIC_IP
+
+  bind ipv4@*:587
+  bind ipv4@*:7777
+
+  default_backend wa_whatsapp_net
+
+backend wa_whatsapp_net
+  default-server check inter 60000 observe layer4
+  server whatsapp_net_443 whatsapp.net:443
```
Comment (slawlor) [inline proxy/src/proxy_config.cfg:94]: I feel like we should update the [healthchecker](https://github.com/WhatsApp/proxy/blob/main/proxy/src/healthcheck.sh) as well?

**Review by slawlor — APPROVED** [review]

File: proxy/src/proxy_config.cfg (hunk 2 above)
Comment (eozturk1) [reply proxy/src/proxy_config.cfg:94]: Done! Let me know what you think!

**Review by Mitsuecom — APPROVED** [review]

**Review by Mitsuecom — APPROVED** [review]

**Review by Mitsuecom — APPROVED** [review]

**Review by Mitsuecom — APPROVED** [review]

**Review by Mitsuecom — APPROVED** [review]

File: proxy/src/proxy_config.cfg (hunk 2 above)
Comment (Zahidosmani) [reply proxy/src/proxy_config.cfg:94]: #217

File: README.md (hunk 1 above)
Comment (Zahidosmani) [reply README.md]: #201

File: README.md [hunk 3]
```diff
@@ -104,6 +104,7 @@ Depending on the scenario in which you utilize your proxy, the proxy container e
...
 
 There are also ports configured which accept incoming [proxy headers](https://www.haproxy.com/blog/use-the-proxy-protocol-to-preserve-a-clients-ip-address/) (version 1 or 2)
```
Comment (Zahidosmani) [inline README.md:109]: haproxy.com/blog/use-the-proxy-protocol-to

**Review by Zahidosmani — COMMENTED** [review]
Thinks

File: README.md (hunk 3 above)
Comment (shoaibmbs1) [reply README.md:109]: Ok

**Review by shoaib9804725 — CHANGES_REQUESTED** [review]
__

**Comment by Rayendarwif** [comment]
Add whatsapp.net frontend

**Comment by Rayendarwif** [comment]
Sya minta Maff Sya nk cuba main thread ini tapi Sya tk reti

Pada Jum, 1 Nov 2024, 1:03 PTG alphachart ***@***.***>
menulis:

**Comment by shoaibmbs1** [comment]
Oo n...vcfyj

**Review by gu-droid — COMMENTED** [review]
gh pr checkout 197

**Review by arrepyos — APPROVED** [review]

**Review by arrepyos — APPROVED** [review]

**Review by tayson121019-creator — APPROVED** [review]

**Pull request merged**